### PR TITLE
add unload warning

### PR DESCRIPTION
**Pull request by vellip**
- [x] debug CKEDITOR change event not firing

### CODE REVIEW

File: meinberlin/apps/contrib/static/js/unload_warning.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+$(function () {
+  var submitted = false
+  var changeHandler = function () {
+    $(window).on('beforeunload', function () {
+      if (!submitted) {
+        return false
+      }
+    })
+  }
```
Comment (xi) [inline meinberlin/apps/contrib/static/js/unload_warning.js]: The spec says that you should set [`event.returnValue`](https://www.w3.org/TR/html5/browsers.html#dom-beforeunloadevent-returnvalue) to a message. [MDN](https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload) says that some browsers ignore the message and always show a default one. I think it does not hurt to add some compatibility code here.

File: meinberlin/apps/contrib/static/js/unload_warning.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  // eslint-disable-next-line no-constant-condition
+  if (!typeof CKEDITOR === 'undefined') {
```
Comment (xi) [inline meinberlin/apps/contrib/static/js/unload_warning.js]: This condition does not work for me. Why not simply `if (CKEDITOR)`?

Comment (vellip) [inline meinberlin/apps/contrib/static/js/unload_warning.js]: It threw an error for me, for some weird reason.

File: meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/base_dashboard_project.html [hunk 3]
```diff
@@ -54,3 +54,8 @@
         </div>
     </div>
 {% endblock %}
+
+
+{% block extra_js %}
+    <script src="{% static 'js/unload_warning.js' %}"></script>
```
Comment (kleingeist) [inline meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/base_dashboard_project.html]: maybe we could just include the  script tag in the body/dashboard_content to ensure it is always loaded. currently templates inheriting from base_dashboard_project.html have to remember to call block.super

Comment (vellip) [reply meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/base_dashboard_project.html]: I really dislike having JS **somewhere** in the middle of `<body>`. I know there are possibly already occurrences where we do this and I know loading it in `<head>` isn't better, but I'd rather not make it worse.

Comment (kleingeist) [reply meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/base_dashboard_project.html]: okay, then i'll merge now :)

**Pull request merged**